Dockerfile

```diff
@@ -23,6 +23,7 @@ ADD . /go/src/github.com/eclipse/che-operator
 RUN case $(uname -m) in \
        x86_64) ARCH="amd64" ;; \
        s390x) ARCH="s390x";; \
+       ppc64le) ARCH="ppc64le";; \
```

Comment:
I might suggest making this less hard-coded and more generic... for s390x, ppc64le, and even future arches...

```
       x86_64) ARCH="amd64" ;; \
       *) ARCH="$1";; \
```

Reply:
why merging instead of having the suggested proper fix for other arches ?